pages/documentation/guidance/accessibility.md

```diff
@@ -6,83 +6,167 @@ redirect_from:
+- Don't rely on hover for critical information
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Sensory-sensitive users
+- Keep it simple
+- Avoid flashing and repetitive animation
+- Avoid very high contrast
+- Avoid alarming colors and images
+- Avoid audio and video that start automatically
+
+## What the design system does
+USWDS provides accessible components, the guidance to implement these components, and the tools to customize and extend the design system accessibly.
+
+{:.border-top-1px.padding-top-2.measure-2}
+### Deliver accessible components
+- Test with screen readers (VoiceOver, NVDA)
+- Test with keyboard-only
+- Test with touch-only
+- Test across browsers and operating systems
+- Collect community feedback
+- Scan with automated tools, like pa11y and aXe, on every code change. If it doesn't pass, we can’t merge the code
+- Manually test components and functionality with a Trusted Tester
+- **If it isn't accessible, it won't ship.**
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Provide guidance on how to implement our components accessibly
+- Include an accessibility section with each component
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Include accessibility-focused tools to build new functionality
+- Colors with magic number contrast
+- Rem-based size tokens and an em-based media query mixin that respond to user scale settings
+- Normalized typography tokens that provide reliable, readable results regardless of the typeface
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Feature smart settings that allow accessible customization with minimal custom code
+- Adding more smart components that change their color based on background
+- Smart settings for links to automatically choose accessible link and hover states
+
+{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
+### Make accessibility one of our core design principles
+- It's not only part of our mission to embrace accessibility in our own work, but to do what we can to encourage it.
+- **We prioritize accessibility issues.** We will work hard to fix accessibility-related issues quickly, and to get them into the next available release. Accessibility is a mission-critical service for USWDS, and we want to know when there's an opportunity to fix something that's broken or improve something that could be better.
+
+While we conduct extensive research during the design and development of components, it is impossible to predict every use case for accessibility and usability overall. That is why we encourage you to [contribute to the community]({{ site.baseurl }}/about/community) by providing feedback on how a component is working on your site. We also recommend performing your own user testing at the page level to gauge the usability of a component within context. We welcome any and all findings and feedback as we work to identify and prioritize improvements to our components and guidance.
```

Comment:
There's an opportunity to link to one of many .gov resources on user testing here, since it's an explicit recommendation: 
- https://digital.gov/topics/usability-testing/
- https://digital.gov/2015/03/19/getting-started-with-usability-testing/
- https://www.plainlanguage.gov/guidelines/test/usability-testing/
- https://www.usability.gov/how-to-and-tools/methods/usability-testing.html

Reply:
@thisisdano @mejiaj Not sure on the call to add Digital.gov, plain language.gov or usability.gov links into this section. Could you weigh in on this and determine if those should be added to this paragraph -- or at least to the Govt A11y Resources section below? Thanks

Reply:
good ideas